students/Sahlberg/Lesson3/Assignment3_StringSlicing.py

```diff
@@ -0,0 +1,68 @@
+"""Ian Sahlberg
+12/20/2018
+Python 210 Assignment 3"""
+
+#Functions
+
+def exchange_first_last(seq):
+    """This returns the first and last item of a sequence exchanged"""
+    first = seq[-1:]
+    mid = seq[1:-1]
+    last = seq[:1]
+    return print(first + mid + last)
+
+def everyOther(val):
+    """This returns a sequence with every other value removed"""
+    print(val[::2])
+
+def foursOut(val):
+    """This returns a sequence with the first and last 4 values removed and every other value in between"""
+    valRange = val[4:-4]
+    return print(valRange[::2])
```

Comment:
* this return will return `None` because `print` returns `None`
* var names should be snake_case -> `val_range`
* you can do this in one line `return val[4:-4:2]`